src/cli/index.js

```diff
@@ -62,6 +66,8 @@ export default (config, http) => {
     console.log(`export ALLOWED_EMAIL_DOMAINS=${conf.emailDomains}`);
     console.log(`export HARVEST_ACCESS_TOKEN=${conf.harvestAccessToken}`);
     console.log(`export HARVEST_ACCOUNT_ID=${conf.harvestAccountId}`);
+    console.log(`export HARVEST_ACCESS_TOKENS=${conf.harvestAccessTokens}`);
```

Comment:
A bit dirty to log clearly secret data (tokens) to logs.

Reply:
CLI module is for local dev usage only, probably purpose there was to get them as decrypted to local env variables for dev purposes. Should be probably some other way in future development, it is not very convenient.